docs/Bioinformatic-Analyses/RNAseq-on-Cavatica/rna_seq_3.md

```diff
@@ -1,39 +1,39 @@
 ====================================
 
-The [Gabriella Miller Kids First Pediatric Data Portal (KF portal)](https://kidsfirstdrc.org) hosts datasets at the intersection of childhood development and cancer from over 16,000 samples with the constant addition of new data.
+The [Gabriella Miller Kids First Pediatric Data Portal (KF Portal)](https://kidsfirstdrc.org) hosts datasets at the intersection of congenital birth defects and pediatric cancers, with genomic files for more than 16,000 participants.
 
 !!! tip "Kids First Data Portal"
 
     Check out our lessons on Kids First to learn more about the [different Data Portal features](../../Common-Fund-Tools/Kids-First/Exploring-Data-in-the-KF-Portal/KF_5_Explore.md) and [building simple to complex queries](../../Common-Fund-Tools/Kids-First/Advanced-KF-Portal-Queries/KF_9_AdvancedQuery.md).
 
-There are data with different access levels hosted on the KF portal including open (processed files, reports, plots, etc) and controlled (raw sequencing files, histological images, etc). For this tutorial, we will use **open access pre-processed files** generated using [Kallisto (v0.43.1)](http://pachterlab.github.io/kallisto//releases/2017/03/20/v0.43.1), which uses pseudoalignments to quantify transcript abundance from raw data.
+Files on the KF Portal are managed through different access levels. _Open access_ files (including processed files of somatic samples) can be viewed and downloaded by any user. _Controlled access_ files (including raw sequencing files and imaging data) require approvals through dbGaP. For this tutorial, we will use **open access pre-processed files** generated using [Kallisto (v0.43.1)](http://pachterlab.github.io/kallisto//releases/2017/03/20/v0.43.1), which uses pseudoalignments to quantify transcript abundance from raw data.
 
-!!! info "KFDRC RNAseq workflow"
+!!! info "KFDRC RNAseq Workflow"
 
-    [Kids First RNAseq pipeline](https://github.com/kids-first/kf-rnaseq-workflow) uses multiple tools/packages for expression detection and fusion calls. The workflow requires raw FASTQ files (controlled access) as input and generates multiple outputs including the Kallisto transcript quantification files. All the output files of this pipeline are available on the portal as open access data. In addition to the restricted data access issue, it is computationally taxing to run this workflow on multiple files.
+    The [Kids First RNAseq Workflow](https://github.com/kids-first/kf-rnaseq-workflow) uses multiple tools/packages for expression detection and fusion calls. The workflow requires raw FASTQ files (controlled access) as input and generates multiple outputs including the Kallisto transcript quantification files. All the output files of this pipeline are available on the KF Portal as open access data. Due to access restrictions and computational intensiveness, this tutorial will not cover this workflow, but users with their own RNAseq data could consider starting here.
```

Comment:
suggestion: "Due to access restrictions and computational intensity, this tutorial will not cover the Kallisto workflow, but users with their own RNAseq data may consider starting with it."